api/config/service/metrics/v1/metrics.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package clutch.config.service.metrics.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/config/service/metrics/v1;metricsv1";
+
+import "validate/validate.proto";
+
+message Config {
+  // This should be an endpoint that mirrors the Prometheus API
+  string prometheus_api_endpoint = 1 [ (validate.rules).string = {min_bytes : 1} ];
```

Comment:
Would these changes to the protos to be close to promethius tie us to promethius?

Reply:
yes. Is that a bad thing?